config/templates/csv-template.yaml

```diff
@@ -6,40 +6,40 @@ metadata:
   annotations:
     categories: A list of comma separated categories that your operator falls under.
     certified: "false"
-    description: GCP project creator. 
+    description: GCP project creator.
     containerImage: quay.io/app-sre/gcp-project-operator:latest
     createdAt: "2020-02-05T16:12:36Z"
     support: Red Hat
 spec:
   displayName: gcp-project-operator
   description: GCP project creator.
   keywords:
-  - kubernetes
-  - openshift
-  - multi-cluster
-  - cluster
+    - kubernetes
+    - openshift
+    - multi-cluster
+    - cluster
   version: 0.0.1
   provider:
     name: Red Hat, Inc
   maturity: alpha
   installModes:
-  - type: OwnNamespace
-    supported: true
-  - type: SingleNamespace
-    supported: true
-  - type: MultiNamespace
-    supported: false
-  - type: AllNamespaces
-    supported: false
+    - type: OwnNamespace
+      supported: true
+    - type: SingleNamespace
+      supported: true
+    - type: MultiNamespace
+      supported: false
+    - type: AllNamespaces
+      supported: false
   install:
     strategy: deployment
     spec:
       clusterPermissions:
-      - serviceAccountName: gcp-project-operator
-        # Rules will be added here by the generate-operator-bundle.py script.
+        - serviceAccountName: gcp-project-operator
+          # Rules will be added here by the generate-operator-bundle.py script.
       deployments:
-      - name: gcp-project-operator
-        # Deployment spec will be added here by the generate-operator-bundle.py script.
+        - name: gcp-project-operator
+          # Deployment spec will be added here by the generate-operator-bundle.py script.
   customresourcedefinitions:
     owned:
     # CRD's will be added here by the generate-operator-bundle.py
```

Comment:
ran a `yq -P -i` and the comment was removed.
could you switch the lines so the next `yq -P` won't show a diff? (or however you linted this yaml)

created an issue for YQ with this example https://github.com/mikefarah/yq/issues/1217

Reply:
Sorry I didn't get you, right `yq -P -i <file>` does remove the last line. Do you want it removed or do you want me to switch lines _44_ & _45_?

``` 
    # CRD's will be added here by the generate-operator-bundle.py
    owned:
```

Reply:
Switch them so for now it'll not be removed